src/index.js

```diff
@@ -18,9 +19,10 @@ try {
 }
 
 // merge default and user configuration
-const config = {
-  ...defaultAppConfig,
-  ...userConfig, // overide with userConfig data
-};
+const config = merge(
```

Comment:
What is the advantage of lodash merge over destructuring assignment ?

Reply:
```
> let defConf = {a:{b:1, c:1}};
undefined
> let userConf = {a:{c:2}};
undefined
> {...defConf, ...userConf};
{ a: { c: 2 } }
> const _ = require('lodash');
undefined
> _.merge({}, defConf, userConf);
{ a: { b: 1, c: 2 } }
```